0.8/docs/devguide/properties.md

```diff
@@ -135,22 +136,27 @@ for more information.
 ## Property name to attribute name mapping {#property-name-mapping}
 
 For data binding, deserializing properties from attributes, and reflecting
-properties back to attributes, {{site.project_title}} must map attribute names to property
-names and the reverse. However, attribute names are case-insensitive and can
-contain dashes, while property names are case-sensitive and cannot contain
-dashes.
+properties back to attributes, {{site.project_title}} maps attribute names to property
+names and the reverse. 
+
+When mapping attribute names to property names:
 
-When mapping attribute names to property names, the general rule is that
-attribute names are converted to lowercase (since the DOM is case-insensitive
-for attribute names).
+*   Attribute names are converted to lowercase property names. For example,
```

Comment:
So it's not a requirement that attribute names contain a dash if their name contains more than one word? I thought in my testing this was the vase. Not so?